packages/docs-site/src/content/docs/resources/terminology.mdx

```diff
@@ -0,0 +1,20 @@
+---
+title: Taiko-specific Terminology
```

Comment:
prefer this to match the filename in kebab case. so can update this to terminology, or the filename to taiko-specific-terminology. also sentence case.